### PR TITLE
Allow installing multiplatform ocp image

### PR DESCRIPTION
**Pull request by tsorya**
This allows to install multiplatform image

### CODE REVIEW

File: support/thirdparty/oc/pkg/cli/image/manifest/manifest.go [hunk 1]
```diff
@@ -186,7 +186,7 @@ func ProcessManifestList(ctx context.Context, srcDigest digest.Digest, srcManife
 		}
 
 		switch {
-		case len(srcManifests) == 1 && !keepManifestList:
```
Comment (alvaroaleman) [inline support/thirdparty/oc/pkg/cli/image/manifest/manifest.go:189]: Is always using the first one correct when len> 1 ?

**Comment by csrwng** [comment]
Thank you!

File: support/thirdparty/oc/pkg/cli/image/manifest/manifest.go (hunk 1 above)
Comment (tsorya) [reply support/thirdparty/oc/pkg/cli/image/manifest/manifest.go:189]: didn't find issue with that yet. At least it doesn't change current behavior and passed all my manual tests

Comment (tsorya) [reply support/thirdparty/oc/pkg/cli/image/manifest/manifest.go:189]: sounds great :)

**Pull request merged**